libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -109,6 +110,14 @@ void AP_Mount::init()
             _backends[instance] = new AP_Mount_Servo(*this, _params[instance], false, instance);
             _num_instances++;
 #endif
+
+#if HAL_MOUNT_SIYI_ENABLED
```

Comment:
Man, we *really* need to change this to a switch at some point!

Reply:
Yes indeed.  a switch would be more consistent with our other front ends..